cmd/kubelet/app/server.go

```diff
@@ -305,7 +305,15 @@ func run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies) (err error) {
 				glog.V(2).Infof("Successfully initialized cloud provider: %q from the config file: %q\n", s.CloudProvider, s.CloudConfigFile)
 			}
 			kubeDeps.Cloud = cloud
+		} else if cloudprovider.IsExternal(s.CloudProvider) {
+			glog.V(2).Infof("WRF - Initting the cloud provider as GCE anyway!!!")
```

Comment:
Is this for temp debugging? If not, what's this for?

Reply:
Debugging. This is a WIP. So far when I generate something which can be split off and used in isolation I am generating a separate PR for that. I may eventually attempt to "clean" this PR but that will be quite a bit of work.